src/algebraic_geometry/Scheme.lean

```diff
@@ -32,19 +32,13 @@ so that that the restriction of `X` to `U` is isomorphic,
 as a locally ringed space, to `Spec.to_LocallyRingedSpace.obj (op R)`
 for some `R : CommRing`.
 -/
-structure Scheme extends X : LocallyRingedSpace :=
-(local_affine : ∀ x : X, ∃ (U : open_nhds x) (R : CommRing),
-  nonempty (X.restrict U.open_embedding ≅ Spec.to_LocallyRingedSpace.obj (op R)))
+structure Scheme extends to_LocallyRingedSpace : LocallyRingedSpace :=
```

Comment:
would it make sense to abbreviate the projection to `to_LRS`?

Reply:
It might even be a good idea to replace `LocallyRingedSpace` with `LRS` everywhere?
Nonetheless there are several places in mathlib that uses `to_LocallyRingedSpace`, so I think this change
(either `to_LocallyRingedSpace` -> `to_LRS` or `LocallyRingedSpace` -> `LRS`) should be done in a separate PR.